src/modules/Popup/Popup.js

```diff
@@ -357,6 +357,11 @@ export default class Popup extends Component {
   handleTriggerRef = (triggerRef) => {
     debug(`handleTriggerRef(${triggerRef})`)
     this.triggerRef = triggerRef
+    // if it is auto popup
+    // then set popup coordinates when triggerref is available
+    if (_.has(this.props, 'open')) {
+      this.setPopupStyle()
+    }
```

Comment:
Should this not only be set if `open` is truthy?  Also, let's link the issue here in the code:

```jsx
handleTriggerRef = (triggerRef) => {
  const { open } = this.props
  // ...

  // https://github.com/Semantic-Org/Semantic-UI-React/issues/2991
  if (open) this.setPopupStyle()
}
```

Reply:
Yes, and we should also add `open` in propTypes list.

Reply:
I'm not sure I understand inherited props. In `popup.d.ts` file we are already inheriting from PortalProps.